ember-cli/2016-07/july-14.md

```diff
@@ -0,0 +1,59 @@
+# ember-cli Meeting Notes - 2016-07-14
+
+## Attendees
+
+- Nathan Hammond
+- Tobias Bieniek
+- Jake Bixby
+- Chad Hietala
+- David Hamilton
+- Stefan Penner
+- Kelly Selden
+- Katie Gengler
+- Robert Jackson
+
+## Actions
+
+- [ ] Nathan - Document merging `[BUGFIX master/beta]` back on a timeline.
+- [ ] Stef - Find GitHub contact.
+- [ ] Tobias - Changelog script updates minimizes milestones.
+- [ ] Stef - RFC/plan for AssetGraph.
+- [ ] Rob - Spike AssetGraph
+- [ ] Tobias - Document plans for babel/eslint.
+
+## Discussion
+
+### Busted release
+
+- broccoli-caching-writer changed its sort method.
+- Previously "undefined" had a consistent order, but not explicitly set.
+- Now has guarantees.
+- The change required proper use of `headerFiles` to play nicely.
```

Comment:
```
The change required proper use of `headerFiles` in broccoli-concat (downstream of BCW) to play nicely.
```

@nathanhammond ^ maybe is a little more clear as orig. makes it sound like `headerFiles` is something you specify to broccoli-caching-writer.